enhance info about CA Chains (#821)
uyuni-project/uyuni-docs: modules/administration/pages/ssl-certs-imported.adoc

Before you begin, ensure you have:

> * A certificate authority (CA) SSL public certificate (Root CA and all intermediate CAs in case a CA chain is used)

Reviewer: ```suggestion
* A certificate authority (CA) SSL public certificate.
  If you are chaining CAs, it must include the root CA, and all intermediate CAs.
```

You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
You can also list multiple host names if your environment requires it.

> It is common that 3rd party Authorities use intermediate CAs to sign requested server certificates. In this case all CAs of the chain are required to be available in 1 file which is used to deploy the CA to the clients.

Reviewer: ```suggestion
Third-party authorities commonly use intermediate CAs to sign requested server certificates. 
In this case, all CAs in the chain are required to be available in a single file to deploy the CA to the clients.
```
